.github/PR_REVIEW_PROCESS.md

```
@@ -0,0 +1,352 @@
+## ✅ Summary
+
+### Before EVERY Merge to `develop`/`qa`/`uat`/`prod`:
+
+1. ✅ Create PR on GitHub
+2. ✅ Tag `@coderabbitai review` in PR description or comment
+3. ✅ Tag `@codex` for architecture review
+4. ✅ Assign `@ak-eyther` as reviewer
+5. ✅ Wait for approvals
+6. ✅ Merge via GitHub UI
+
+### NEVER:
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)